solutions/fully-configurable/DA-types.md

```diff
@@ -0,0 +1,125 @@
+
+### Options for cloud_logs_policies
+
+  - `logs_policy_name` (required): The unique policy name.
+  - `logs_policy_description` (optional): The description of the policy to create.
+  - `logs_policy_priority` (required): The priority to determine the pipeline for the logs. Allowed values are: type_unspecified, type_block, type_low, type_medium, type_high. High (priority value) sent to 'Priority insights' (TCO pipleine), Medium to 'Analyze and alert', Low to 'Store and search', Blocked are not sent to any pipeline.
+  - `application_rule` (optional): The rules to include in the policy configuration for matching applications.
+  - `subsystem_rule` (optional): The subsystem rules to include in the policy configuration for matching applications.
+  - `log_rules` (required): The log severities to include in the policy configuration.
+  - `archive_retention` (optional): Define archive retention.
+
+### Example cloud_logs_policies
+
+```hcl
+cloud_logs_policies = [
+  {
+    logs_policy_name     = "logs-policy-1"
+    logs_policy_description = "Send info and debug logs of the application (name starts with `test-system-app`) and the subsytem (name starts with `test-sub-system`) logs to Store nad search pipeline"
+    logs_policy_priority = "type_low"
+    application_rule = [{
+      name         = "test-system-app"
+      rule_type_id = "start_with"
+    }]
+    log_rules = [{
+      severities = ["info", "debug"]
+    }]
+    subsystem_rule = [{
+      name         = "test-sub-system"
+      rule_type_id = "start_with"
+    }]
+  },
+  {
+    logs_policy_name     = "logs-policy-2"
+    logs_policy_description = "Send error logs of all applications and all subsystems to Analyze and Alert pipeline"
+    logs_policy_priority = "type_medium"
+    log_rules = [{
+      severities = ["error"]
+    }]
+  }
+]
+```
+
+
+## cloud_log_data_bucket_retention_policy <a name="cloud_log_data_bucket_retention_policy"></a>
+
+The `cloud_log_data_bucket_retention_policy` input variable allows you to provide the retention policy of the IBM Cloud Logs data bucket that will be configured. Refer [here](https://cloud.ibm.com/docs/cloud-object-storage?topic=cloud-object-storage-immutable) for more information.
+
+- Variable name: `cloud_log_data_bucket_retention_policy`.
+- Type: An object representing a retention policy.
+- Default value: null (`null`).
+
+### Options for cloud_log_data_bucket_retention_policy
+
+- `default` (optional): The number of days that an object can remain unmodified in an Object Storage bucket.
+- `maximum` (optional): The maximum number of days that an object can be kept unmodified in the bucket.
+- `minimum` (optional): The minimum number of days that an object must be kept unmodified in the bucket.
+- `permanent` (optional): Whether permanent retention status is enabled for the Object Storage bucket.
+
+### Example cloud_log_data_bucket_retention_policy
+
+```hcl
+cloud_log_data_bucket_retention_policy = {
```

Comment:
The code block should not include `cloud_log_data_bucket_retention_policy =` in it. It should just include the value of the variable so users can click the copy to clipboard in the UI and paste the exact value